### DISCUSSION TITLE
Use windows brightness buttons as Hotkeys for all Monitors

### DISCUSSION THREAD

**Discussion by TheGoldenMan99** [xanderfrangos/twinkle-tray#771] [Ideas]
Currently, when selection a hotkey in twinkley for brightness control it is not possible to set the brightness buttons of my keyboard, because they directly change the brightness instead. It would be very usefull if I could use them as twinkley hotkeys, or if twinkley could simply sync to the standard windows brightness buttons.

**Comment by mehmetakyurek** [comment]
I think in order to do that we need to unbind those keys from Windows. I checked and couldn't find a way to do it. None of the apps can detect those key presses so it might be that Windows handles these keys on lower level and none of the apps can detect those keys.
